lightly/utils/io.py

```diff
@@ -91,6 +94,18 @@ def check_embeddings(path: str):
                     f'Found empty row on line {i}.'
                     )
 
+    if remove_additional_columns:
+        new_rows = []
+        with open(path, 'r', newline='') as csv_file:
+            reader = csv.reader(csv_file, delimiter=',')
+            for row in reader:
+                row = row[:header_labels_idx+1]
```

Comment:
This assumes that the first columns always contain (filenames, embeddings, labels) and no one added another column in between. Is this assumption always correct? Or should we add a check to verify this?

If this is the assumption we take please state it in the docstring. Because right now I would assume that other columns are dropped regardless of their position.

Reply:
This is not assumed, but checked for in the lines above.